webapp/src/index.tsx

```diff
@@ -615,17 +616,19 @@ export default class Plugin {
                             },
                         });
 
-                        const callExists = incomingCalls(store.getState()).findIndex((ic) => ic.callID === data[i].channel_id) >= 0;
-                        if (data[i].call && !callExists) {
-                            // dismissedNotification is populated after the actions array has been batched, so manually check:
-                            const dismissed = data[i].call?.dismissed_notification;
-                            if (dismissed) {
-                                const currentUserID = getCurrentUserId(store.getState());
-                                if (Object.hasOwn(dismissed, currentUserID) && dismissed[currentUserID]) {
-                                    continue;
+                        if (ringingEnabled(store.getState())) {
```

Comment:
I realize we can continue early here, but I think that would be more confusing, and prone to mistakes later. Here (and elsewhere) we're just saying: if ringing is enabled, do the following).